### ISSUE TITLE
Function to extract the statistics of the underlying distribution 

### ISSUE THREAD

**Issue by AIX2** [rte-france/Grid2Op#152]
## Is your feature request related to a problem? Please describe.
Yes, I was going to extract the statistics of the underlying distribution in the Neurips track2 environment. 

## Describe the solution you'd like
Functions to extract mean and standard deviation of loads, prods etc. 

## Describe alternatives you've considered
I can run grid2op runner to calculate them, however an internal function save me time.  

## Additional context
Thank you!

**Comment by BDonnot** [comment]
The new grid2op (that will be released in next grid2op version, 1.3.2) does that.

This module is called `utils` and will take care of storing some running of past episodes directly in the environment (note that the storing is a trade off between space taken and ease of access, soo it will take quite some space if you run lots of agents on lots of scenarios...)

In a nutshell you will be able to use it this way:
```python
        import grid2op
        from grid2op.utils import EpisodeStatistics
        env = grid2op.make()

        stats = EpisodeStatistics(env)

        #################################
        # This need to be done only once
        stats.compute(nb_scenario=100)   # this will take a while to compute in most cases
        ################################

        rhos_, scenario_ids = stats.get("rho")
        load_p_, scenario_ids = stats.get("load_p")

        # do something with them
```

Documentation will be updated when the package will be out

**Comment by BDonnot** [comment]
New version 1.4.0 now address this issue and makes the above code possible :-)
